DSCResources/MSFT_xDnsARecord/MSFT_xDnsARecord.psm1

```diff
@@ -23,7 +23,7 @@
     Write-Warning -Message "DSC Resource xDnsARecord has been replaced by xDNSRecord, and will be removed in a future version"
     Write-Verbose "Looking up DNS record for $Name in $Zone"
     $record = Get-DnsServerResourceRecord -ZoneName $Zone -Name $Name -ErrorAction SilentlyContinue
-    if ($record -eq $null) {
+    if ([bool]($record -eq $null)) {
```

Comment:
The cast to a boolean isn't needed here.
Also the `$null` should be on the left side of the comparison instead of the right.
You can read more about it here: https://github.com/PowerShell/PSScriptAnalyzer/blob/development/RuleDocumentation/PossibleIncorrectComparisonWithNull.md